packages/server/lib/gui/auth.ts

```diff
@@ -24,7 +24,7 @@ const buildLoginRedirectUrl = (server) => {
   return `http://127.0.0.1:${port}/redirect-to-auth`
 }
 
-const buildFullLoginUrl = (baseLoginUrl, server, utmSource, utmMedium) => {
+const buildFullLoginUrl = (baseLoginUrl, server, utmSource, utmMedium, utmContent) => {
```

Comment:
Nit: last three args should be an object of options, like `utmData`. Otherwise, it's easy to mix them up since they are all strings.

Reply:
I like objects in general when there's multiple params though typescript helps a bunch. I'd agree not essential to change this, just a preference.